src/components/molecules/SecretPasswordForm/SecretPasswordForm.jsx

```diff
@@ -1,12 +1,10 @@
 import React, { useState } from "react";
-import { useParams, useHistory } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { useFirebase } from "react-redux-firebase";
 import "./SecretPasswordForm.scss";
-import { venueEntranceUrl } from "utils/url";
 
-const SecretPasswordForm = ({ buttonText = "Join the party" }) => {
+const SecretPasswordForm = ({ buttonText = "Join the party", onSuccess }) => {
```

Comment:
On what success? This could be named better to convey its meaning/usage.

Also, where are the prop types for this component??

```suggestion
export interface SecretPasswordFormProps {
  buttonText?: string
  onSuccess?: () => void
}

const SecretPasswordForm: React.FC<SecretPasswordFormProps> = ({ buttonText = "Join the party", onSuccess }) => {
```